src/universal/styles/ui.js

```diff
@@ -360,7 +355,7 @@ const ui = {
   menuBackgroundColor: '#fff',
   menuBorderColor: appTheme.palette.mid30l,
   menuBorderRadius: borderRadiusSmall,
-  menuBoxShadow: shadow[2],
+  menuBoxShadow: shadow[3],
```

Comment:
what happens if we have a dropdown menu on a modal? would we have a level 3 shadow on a level 4 element?

Reply:
yeah i think i'll take that on, but let's merge this into 6.5 & get it off your personal branch & then i can do some edits from there